### DISCUSSION TITLE
issue: field removed after delete information in input

### DISCUSSION THREAD

**Discussion by olegvrs** [react-hook-form/react-hook-form#9419] [General]
Hello! I have a problem. I use setValue and update. I have a button (onClick(() => setValue)) which adds a character ('/') to the text in the input. 
Example: text1 / text2 / text3 / text4 / text5.

After entering the text (f.e. text2), I click on the button and a character is added after the entered text (setValue(`list.${index}.${step}`, fields[index][step] + ' / ')). 
On the form is onChange(() => update(fields[index][step]). In order for there to be an instant update of the input.

But if I start removing the text completely, the shape disappears. 
How can I avoid the field disappearing? Pls help me what i need to do so that the field is not deleted.
Perhaps I explained the problem incorrectly, but I think it's clear. My english is very poor excuse me.

**Comment by bluebill1049** [comment]
share a simple coesandbox? when you set a object field, you should carry the other existing attributes.
